integrations-and-supported-tools/transformers/notebooks/Neptune_Transformers.ipynb

```diff
@@ -0,0 +1,382 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "header",
+     "comment"
+    ]
+   },
+   "source": [
+    "# Using the 🤗 Transformers Integration\n",
+    "\n",
+    "Neptune provides an integration with 🤗 Transformers. So all you need to do to log metadata of the 🤗 Transformers training/finetuning is write a few lines of additional code and logging to Neptune is taken care of. \n",
+    "\n",
+    "You can integrate metadata tracking with Neptune either by:\n",
+    "* passing report_to = \"neptune\" to the Trainer arguments.\n",
+    "* setting up a Neptune callback and passing it to the Trainer callbacks.\n",
+    "\n",
+    "In this guide, we will look at the second option i.e. setting up a Neptune callback and passing it to the Trainer.\n",
+    "\n",
+    "By the end of this guide, you will be able to use the HuggingFace Transformers integration to log\n",
+    "* Train Loss\n",
+    "* Evaluation Loss\n",
+    "* Trainer parameters\n",
+    "* Model parameters\n",
+    "* Model checkpoint\n",
+    "\n",
+    "[See this example in Neptune](https://app.neptune.ai/o/common/org/quickstarts/e/QUI-81820/charts)\n",
```

Comment:
Lets use an example from [showcase/project-text-summarization-hf](https://app.neptune.ai/o/showcase/org/project-text-summarization-hf/experiments?split=tbl&dash=charts&viewId=standard-view). Also remember to tag the selected run as a `showcase-run`